packages/minify/index.js

```diff
@@ -1,5 +1,8 @@
-const BabelMinify = require('babel-minify-webpack-plugin');
+const babelMinify = require('@neutrinojs/babel-minify');
+const imageMinify = require('@neutrinojs/image-minify');
 
-module.exports = ({ config }, options = {}) => config
-  .plugin('minify')
-  .use(BabelMinify, [options.minify, options.overrides]);
+module.exports = (neutrino, options = {}) => {
+  neutrino.config
```

Comment:
Maybe this would be easier by breaking it into merged options:

```js
const options = merge({
  babel: opts.babel ? {} : false,
  image: opts.image ? {} : false
}, opts);

neutrino.config
  .when(options.babel, () => neutrino.use(babelMinify, options.babel))
  .when(options.image, () => neutrino.use(imageMinify, options.image));
```

Thoughts?

Reply:
Probably. Easier to read.

Reply:
Hmmm that would still just end up with values as `true` though

Reply:
Oh derp.

Reply:
*nothing to see here*

Reply:
Do you think we should care about this? Do we want to avoid users "enabling" by passing `{}`?

Reply:
No, passing an empty object should be a legit way to enable this. I think you were on the right track, passing `true` should coerce to object, and then merge the options.

Reply:
> Do we want to avoid users "enabling" by passing {}?

Sorry I meant do we want to avoid that being the _only_ way of enabling (aside from actually passing options).

Reply:
Could you elaborate?

Reply:
`{}` will always be a legit way of enabling. Just verifying that we _want_ to bother coercing `true` values to `{}`.

Reply:
Ah I see. Yeah, when you think about it from an enabled standpoint, using `true` doesn't make a whole lot of sense when used with an item that starts off in an enabled state. If `minify` already has both items enabled, then passing true doesn't seem relevant. So I guess my answer would be don't bother with `true` from the `minify` middleware.

From the perspective of the `web` middleware though, `images` would start off disabled, so maybe it's `web` that allows a `true` option instead.

Reply:
Sounds good - I'll coerce there instead.